slurm_ops_manager/slurm_ops_base.py

```diff
@@ -296,7 +297,12 @@ def create_configless_systemd_override(self, host, port):
 
         environment = Environment(loader=FileSystemLoader(self._template_dir))
         template = environment.get_template("configless.default.tmpl")
-        target = Path("/etc/default/slurmd")
+
+        if operating_system() == 'ubuntu':
+            target = Path("/etc/default/slurmd")
+        else:
+            target = Path("/etc/sysconfig/slurmd")
```

Comment:
If we are covering only ubuntu and centos7, maybe shows an error if not ubuntu nor centos?
```suggestion
        if operating_system() == 'ubuntu':
            target = Path("/etc/default/slurmd")
        elif operating_system() == 'centos':  # maybe centos in operation_system() is a better approach better, same for ubuntu
            target = Path("/etc/sysconfig/slurmd")
        else:
            raise NotImplementedError
```

Reply:
We already have many checks to ensure the correct OS before this code.

Reply:
And what about changing directly string comparison with the suggested one?